modules/xmpp/ChatRoom.js

```diff
@@ -1113,10 +1095,10 @@ export default class ChatRoom extends Listenable {
         if (txt) {
             if (type === 'chat') {
                 this.eventEmitter.emit(XMPPEvents.PRIVATE_MESSAGE_RECEIVED,
-                        from, nick, txt, this.myroomjid, stamp);
+                        from, txt, this.myroomjid, stamp);
```

Comment:
Won't this require changes in Jitsi Meet?

Reply:
Nope these are the xmpp events internal to the lib and the change is done

Reply:
Great!